.gitignore

```diff
@@ -10,6 +10,8 @@
 build
 sklearn/datasets/__config__.py
 sklearn/**/*.html
+.env
```

Comment:
I'm happy with this change, but it needs its own PR

Reply:
Indeed, two PRs got mixed. We fixed